tidy: smooth logo swoosh

alexallmont opened the pull request.

alexallmont committed:
tidy: smooth logo swoosh

martinjrobins committed:
Merge branch 'main' into tidy-logo

martinjrobins commented:
the logo looks great, is this ready to merge or are you still working on it?

alexallmont committed:
tidy: further smoothing on rect logo

alexallmont commented:
> the logo looks great, is this ready to merge or are you still working on it?

@martinjrobins I've made one further tweak, do you think it's improved it? Please let me know if so and I will apply changes to the other logos.

martinjrobins commented:
yes, this looks great :) please edit the others as well and we can get this merged

alexallmont committed:
tidy: smooth remaining logo shapes

- Applied rect changes to hex and square.
- Note for future edits, it's easiest to tweak the shape in the square version as that has a circular border - better reference for fitting curve.

alexallmont commented:
@martinjrobins I've updated other shapes, ready to merge if you're happy with this.

A note for future edits here: it's easiest to use the 'square' version as the base logo to work from because that has a circular border for reference against curve.

martinjrobins reviewed (approved):
looks great, thanks @alexallmont
